libraries/load-themed-styles/src/index.ts

```diff
@@ -103,6 +103,9 @@ let _injectStylesWithCssText: boolean;
 // load-themed-styles hosted on the page.
 const _root: any = (typeof window === 'undefined') ? global : window; // tslint:disable-line:no-any
 
+// Nonce string to inject into script tag if one provided. This is used in CSP (Content Security Policy).
+let _styleNonce: string = _root && _root.CSPSettings && _root.CSPSettings.nonce;
```

Comment:
This should work, but also I think we should update the README.md and include documentation for this setting.

There's also an existing tests file. It would be good to add a test so nobody breaks it in the future.

Reply:
Added a snippet to readme. The test requires a dom-like environment. It does not seem one is configured so the code never gets to place where style element is created.